electrumsv/wallet_database/functions.py

```diff
@@ -640,9 +640,10 @@ def read_history_for_outputs(db: sqlite3.Connection, account_id: int, *,
     INNER JOIN TransactionOutputs TXO ON TXO.tx_hash=ATX.tx_hash
     LEFT JOIN _AccountSpends _AS ON _AS.tx_hash=ATX.tx_hash
     LEFT JOIN KeyInstances KI ON KI.keyinstance_id=TXO.keyinstance_id
+    --TXO.keyinstance_id IS NULL indicates that it's paying an external wallet
```

Comment:
When you add a comment, please consider if you have written why something matters. Please change and say why this matters. Just making an assertion about a fact you consider important leaves no context for other people to understand why.

Reply:
Removed the comment.